src/main/java/org/fcrepo/spec/testsuite/versioning/AbstractVersioningTest.java

```diff
@@ -56,10 +57,18 @@ protected URI getTimeMapUri(final Response response) {
     protected Response createVersionedResource(final String uri, final TestInfo info) {
         final Headers headers = new Headers(
             new Header("Link", ORIGINAL_RESOURCE_LINK_HEADER),
-            new Header(SLUG, info.getId()));
```

Comment:
I don't think we need this unless someone has a good reason to have a known name for a resource.  It makes it hard to run the test multiple times w/o clearing the DB in between.  It's a larger picture question because slugs are used in many spots in the code.

Reply:
Agreed. Giving test resources fixed names is not ideal.